src/app/pages/top-up/top-up.component.html

```diff
@@ -0,0 +1,98 @@
+<div class="p-safe h-svh overflow-auto">
+  <div class="flex h-full flex-col items-start justify-start">
+    <div class="reduced-page-container w-full p-4">
+      <rusbe-header
+        [customAction]="goToPreviousStage.bind(this)"
+        [type]="headerType()"
+      >
+        ADICIONAR CRÉDITOS
```

Comment:
nit: Header component automatically transforms text to be uppercase

Reply:
Updated.